doc/en/how-to/fixtures.rst

```diff
@@ -735,8 +735,78 @@ does offer some nuances for when you're in a pinch.
 .. code-block:: pytest
 
    $ pytest -q test_emaillib.py
-   .                                                                    [100%]
-   1 passed in 0.12s
+  .                                                                    [100%]
+  1 passed in 0.12s
+
+Note on finalizer order
+""""""""""""""""""""""""
+
+Finalizers are executed in a first-in-last-out order, while operations after yield are executed sequentially.
```

Comment:
>  while operations after yield are executed sequentially.

This sentence needs tweaking if we look at the output. 👍 

Finalizers are executed in FILO order, that is correct, because we can see that adding `fin2` and `fin1` we get back `fn1`, `fn2` in the output.

But the same can be said for the yield fixtures: we request them in `fix_w_yield1`, `fix_w_yield2` order, and we get `fix_w_yield2` and `fix_w_yield1` in the output.

That's because `yield` fixtures use `addfinalizer` behind the scenes: when the fixture executes, we register a function with `addfinalizer` that resumes the generator (which in turn calls the teardown code).

Not sure how to "fix" this sentence, or perhaps we can explain something along the lines of what I wrote above, if that makes things clearer.

Reply:
Yeah, that makes sense.
I tried to explain how both examples show the similar FILO behavior, plus added the hidden usage of addfinalizer after the examples.
What do you think?